osm2gtfs/creators/trips_creator.py

```diff
@@ -166,16 +184,16 @@ def _add_itinerary_trips(self, feed, itinerary, line, trip_builder,
             gtfs_trip = route.AddTrip(feed, headsign=itinerary.to,
                                       service_period=trip_builder['service_period'])
             trips_count += 1
+            search_idx = 0
```

Comment:
To prevent problems in `itineraries` where a stop (or a stop_name) is served twice, so that it doesn't pick the same time information twice (this would give "high-speed services" for the transitfeed validator), I've introduced the index from where it should search for the next time.

Reply:
I tested with Managua and it is the same result. Generally checked the code and looks good.